erizo_controller/erizoClient/src/Erizo.js

```diff
@@ -3,14 +3,20 @@ import { LicodeEvent, RoomEvent, StreamEvent } from './Events';
 import Stream from './Stream';
 import Logger from './utils/Logger';
 
-// eslint-disable-next-line 
-require('expose-loader?adapter!../lib/adapter.js');
-// eslint-disable-next-line
-require('script-loader!./utils/L.Resizer.js');
+// #if process.env.INCLUDE_ADAPTER == 'TRUE'
```

Comment:
Why not make it the opposite and remove the import when a variable is present? I'd prefer the default behavior to stay the same